src/storage/v3/conversions.hpp

```diff
@@ -69,6 +69,10 @@ TTypedValue PropertyToTypedValue(const PropertyValue &value) {
   LOG_FATAL("Unsupported type");
 }
 
+template <typename TypedValueT>
+inline const auto PropertyToTypedValueFunctor =
+    [](const PropertyValue &value) { return PropertyToTypedValue<TypedValueT>(value); };
+
```

Comment:
Isn't `PropertyToTypedValueConverter` does the same?

Reply:
This is not addresssed.

Reply:
This is not addressed.

Reply:
As discussed these two functions are not identical (the one in the bindings contain an extra overload) so we decided to keep it as is.